util/index.js

```diff
@@ -8,3 +12,37 @@ export function preprocessJsonInput(text) {
   }
 }
 export {parseSite} from './page-parser.js';
+
+export function createTestFile(filePath) {
+  const boilerPlate = `import { test, expect } from '@playwright/test';
+
+test('generated test', async ({ page }) => {
+`;
+
+  fs.writeFileSync(filePath, boilerPlate.trim() + '\n');
+}
+
+export function appendToTestFile(userInput, generatedCode, filePath) {
+  let formattedCode = `\t// ${userInput}\n`;
+
+  // Split the code into lines and format each line
+  const lines = generatedCode.split('\n');
+  for (const line of lines) {
+    formattedCode += `\t${line.trim()}\n`;
+  }
+
+  fs.appendFileSync(filePath, `${formattedCode}\n`, 'utf8');
+}
+
+export function completeTestFile(filePath) {
+  fs.appendFileSync(filePath, '});', 'utf8');
+}
+
+export function gracefulExit(options) {
+  if (options.outputFilePath) {
+    completeTestFile(options.outputFilePath);
+  }
+
+  console.log('Exiting'.red);
+  process.exit(0);
```

Comment:
you dont need to call exit explicitly here.

Reply:
Ahh okay, good to know

Reply:
https://github.com/mayt/BrowserGPT/pull/16/commits/11662d38e76ef4ed6433c3a9a24020ef9299c543